### DISCUSSION TITLE
How do I search all drives?

### DISCUSSION THREAD

**Discussion by endolith** [dnGrep/dnGrep#1054] [Q&A] [answered]
If I set the path to `C:\Users\` it searches across 13222 files, but if I set the path to `C:\` it searches across 0 files.  If I set it to `C:\;D:\;F:\` it searches across 6554 files, but they all seem to be on the F drive.  I don't understand how to get it to search across all drives and subfolders.

**Comment by doug24** [comment, marked as answer]
Do you have the `Include hidden folders` unchecked?  If so, try checking it.

The root of the C:\ drive is marked as a hidden folder - it is a hidden system directory.  There is code in dnGrep to skip checking for the hidden attribute when it is the root of the drive, but it looks like the implementation of the check for the drive root changed and now that code is broken.  I can fix it in the next release.

If you do have the `Include hidden folders` checked and it's still not working, attach a screenshot of your main window with all the file filters.
